packages/zkdb/src/sdk/impl/collection.ts

```diff
@@ -221,25 +221,14 @@ export class CollectionQueryImpl implements ZKCollection {
   }
 
   async setPermissions(permissions: Permissions): Promise<Ownership> {
-    const remotePermissions = await this.getOwnership();
-
     const result = await this.apiClient.permission.set({
       databaseName: this.databaseName,
       collectionName: this.collectionName,
       docId: undefined,
       permission: {
```

Comment:
does Permissions type that satisfies all permission from the result, If yes you can pass a whole obj 
```ts
const result = await this.apiClient.permission.set({
      databaseName: this.databaseName,
      collectionName: this.collectionName,
      docId: undefined,
      permission
      })
```